ttnn/cpp/ttnn/operations/moreh/moreh_clip_grad_norm/moreh_clip_grad_norm_step2/device/kernels/moreh_clip_grad_norm_step2_kernel.cpp

```diff
@@ -40,38 +40,37 @@ void MAIN {
     // Compute cb_x
     for (uint32_t tile_idx = 0; tile_idx < num_tiles; tile_idx++) {
         if (tile_idx == 0) {
-            ACQ();
+            tile_regs_acquire();
             cb_wait_front(cb_input, onetile);  // comes from the reader
-            cb_reserve_back(cb_x, onetile);
```

Comment:
Same thing above

Reply:
resolved in [ce8237a](https://github.com/tenstorrent/tt-metal/pull/15897/commits/ce8237a7c45fd186600377fcd4f57975e5e5fd4e)